### ISSUE TITLE
Clean HDMI out

### ISSUE THREAD

**Issue by diogofmmartins** [ge0rg/samsung-nx-hacks#18]
Hi guys.
I have seen this amazing project you have here modifying the NX system firmware.

I have no experience in programming but i've been working with the NX system for a while.

I wanted to do a very simple (i think) mod on my Nx300, but I am not sure how or if it is even possible.

Basically, the NX system has an micro HDMI out on the hardware that can only be used to preview media.
I wanted to be able to simply connect an HDMI cable to it and that it would do a CLEAN signal from the camera. This would mean I could connect it to the video mixier and have a clean signal to use.

The magiclantern guys have been doing this simple feature to the old canons like the 500D or 550D for a while and I Imagine it shouldn't be that hard. Or maybe I am wrong.. haha

Can anyone help me?

Thanks!

**Comment by DayVeeBoi** [comment]
I don't think this project progressed that far and it hasn't had any activity in a while. There's a similar project with nx500/nx1 and someone asked a question similar to your request, and the owner of the other project closed the issue with the comment "Not Possible (from what we know)".
https://github.com/ottokiksmaler/nx500_nx1_modding/issues/83

**Comment by ottokiksmaler** [comment]
I'm the maintainer of the other repo. As far as I know this possibility does not exist for NX300. NX500 can output clean 1080p and NX1 can output 4k.

**Comment by DayVeeBoi** [comment]
I apologize for my (probably) correct (but for the wrong reasons) answer.

**Comment by amirc2013** [comment]
> I'm the maintainer of the other repo. As far as I know this possibility does not exist for NX300. NX500 can output clean 1080p and NX1 can output 4k.

What about nx mini ?

**Comment by mxa** [comment]
Oh, that's sad, I was hoping to revive this camera as a deluxe webcam for videoconferencing.

**Comment by pinguinpfleger** [comment]
Did someone already analyzed the processes running on the camera when hdmi cable got connected?

**Comment by alenaester** [comment]
I was looking for that topic, but for the NX3000. I used a capture card from aliexpress, but there is no image, I am looking for some way to be able to create this live view.

**Comment by emanuele** [comment]
> I'm the maintainer of the other repo. As far as I know this possibility does not exist for NX300. NX500 can output clean 1080p and NX1 can output 4k.

@ottokiksmaler how to get clean HDMI output from NX500? After trying all menus still some icons/text remains on the HDMI output...

**Comment by muba1** [comment]
any chance to get clean image live feed stream from nx300m even with an hdmi capture card ? and why it is only for nx500, nx30 and nx1 but not others ?  any firmware update or hacking about that issue?

**Comment by SohnGottesITA** [comment]
Is there a solution to have live streaming with HDMI?
Samsung NX300

**Comment by ElrohiFilmon** [comment]
NX11, I know but I'm taking a huge gamble and either a firmware hack or a custom firmware pipeline. I'm currently taking assembly this term with 8086 basics and I feel that I might puch this project. And also would love to get pointer on where I can find the firmware etc...
